common/src/main/res/values/strings.xml

```diff
@@ -1382,4 +1382,10 @@
     <string name="thermostat_tile">Thermostat tile</string>
     <string name="climate_heating">Heating</string>
     <string name="climate_cooling">Cooling</string>
+    <string name="launcher">Launcher</string>
+    <string name="launcher_option_title">Enable Home Assistant launcher mode</string>
```

Comment:
"launcher mode" suggests there are other changes in the app to me. I was thinking of "Available as launcher app" but that may be too abstract without reading the description.